src/pages/ReportSettingsPage.js

```diff
@@ -96,6 +96,11 @@ class ReportSettingsPage extends Component {
     validate() {
         const errors = {};
 
+        // We return false because because we don't want to perform any action
```

Comment:
The comment should explain the code or technical reason behind that change. These kinds of comments are not useful. 

We should change it to: e.g.
`When the report name is not changed, skip the form submission. Add check here to keep the code clean`.